SoomlaiOSProfile/UserProfileUtils.h

```diff
@@ -22,7 +22,8 @@
 typedef enum {
     FACEBOOK = 0,
     GOOGLE = 2,
-    TWITTER = 5
+    TWITTER = 5,
+    GAMECENTER = 7
```

Comment:
@EugeneButusov @vedi make it 13 (we had other providers until 11, and 12 is currently saved for someone)